packages/x-charts/src/RadarChart/RadarChart.tsx

```diff
@@ -18,8 +18,8 @@ export interface RadarChartSlotProps {}
 
 export interface RadarChartProps
   extends RadarDataProviderProps,
-    RadarGridProps,
-    Partial<RadarAxisHighlightProps>,
+    Omit<RadarGridProps, 'classes'>,
+    Omit<Partial<RadarAxisHighlightProps>, 'classes'>,
     Omit<ChartsOverlayProps, 'slots' | 'slotProps'> {
```

Comment:
We will need to find a way to propagated nested classes.

I'm thinking about something like that

```js
<RadarChart
  classes={{
    grid: { ... },
    axisHIghlight: { ... }
  }}
```